command/assets/Format.go

```diff
@@ -0,0 +1,19 @@
+package assets
+
+import (
+	"math"
+	"strconv"
+	"unicode/utf8"
+)
+
+func PaddingRight(val string, length int, text string) string {
+	for ; utf8.RuneCountInString(val + text) <= length; {
```

Comment:
```suggestion
	for utf8.RuneCountInString(val + text) <= length {
```
https://blog.y-yuki.net/entry/2017/05/06/000000

Reply:
フォーマットかけてませんでしたとても非常にごめんなさい
あと記事ありがとー！